pkg/kube/vault/vault.go

```diff
@@ -103,6 +125,7 @@ type Telemetry struct {
 // Storage configuration for Vault storage
 type Storage struct {
 	GCS GCSConfig `json:"gcs"`
+	DynamoDB DynamoDBConfig `json:"dynamodb"`
```

Comment:
I think you need to add `omitempty` to use these configs independently.

Reply:
Yes, but I think I also need to make GCS and DynamoDB pointers:
https://www.sohamkamani.com/blog/golang/2018-07-19-golang-omitempty/#values-that-cannot-be-omitted

Now I remember: That was what I was supposed to fix this morning before creating the pull request ;)